### PR TITLE
Update support for Boundary Devices

### PR DESCRIPTION
**Pull request by chrisdimich**

### CODE REVIEW

File: meta-mender-nxp/README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ The supported and tested boards are:
  - [WaRP7](https://hub.mender.io/t/nxp-warp7/135)
  - [Pico-Pi i.MX7D](https://hub.mender.io/t/technexion-pico-pi-imx7/136)
  - [Nitrogen8M](https://hub.mender.io/t/boundary-devices-nitrogen8m/409)
+ - [Nitrogen8MM](https://hub.mender.io/t/boundary-devices-nitrogen8m/409)
  - [i.MX7D SABRE](https://hub.mender.io/t/nxp-i-mx7d-sabre/1279)
```
Comment (texierp) [inline meta-mender-nxp/README.md:13]: You missed to upgrade the dependencies here : https://github.com/mendersoftware/meta-mender-community/blob/dunfell/meta-mender-nxp/README.md#dependencies

**Review by texierp — CHANGES_REQUESTED** [review]
Thanks for this update @chrisdimich :)

Just one small things about dependencies

File: meta-mender-nxp/README.md (hunk 1 above)
Comment (chrisdimich) [reply meta-mender-nxp/README.md:13]: Thanks for the review! I will make this change and update the PR.

**Comment by chrisdimich** [comment]
@texierp I have fixed the README. One question, is it preferred to have a separate sublayer called "meta-mender-boundary" or shall we keep the boundary implementation in meta-mender-nxp?

**Comment by chrisdimich** [comment]
Can this please be pushed through? We would like to start advertising Boundary devices with Mender :)

**Comment by texierp** [comment]
> @texierp I have fixed the README. One question, is it preferred to have a separate sublayer called "meta-mender-boundary" or shall we keep the boundary implementation in meta-mender-nxp?

I think you're right, seems better to create a sublayer ;)

**Comment by texierp** [comment]
> Can this please be pushed through? We would like to start advertising Boundary devices with Mender :)

LGTM

**Review by texierp — APPROVED** [review]

**Comment by TheMeaningfulEngineer** [comment]
Thanks for the contribution, looks good to me.
Merging

**Pull request merged**